packages/editor/src/components/collab-sidebar/comments.js

```diff
@@ -126,18 +128,28 @@ function Thread( {
 	};
 
 	const clearThreadFocus = () => {
-		setFocusThread( null );
+		setSelectedThread( null );
 		setShowCommentBoard( false );
 	};
 
 	return (
 		<VStack
 			className={ clsx( 'editor-collab-sidebar-panel__thread', {
-				'editor-collab-sidebar-panel__focus-thread': isFocused,
+				'is-selected': isSelected,
 			} ) }
-			id={ thread.id }
```

Comment:
Maybe we should keep the `id` around, but make it mess generic - `thread-${ thread.id }`

Reply:
Makes sense. Restored in 04cce97c73a724e9fb37da75800ad5b06293e2f5